src/models/collection-class.js

```diff
@@ -0,0 +1,57 @@
+//routing and crud functionality in this file.
+'use strict';
+
+class Collection {
+    constructor(name, model) {
+        this.model = model;
+        this.name = name;
+    }
+
+    read(id, options = {}) {
+        let modelParams = { ...options };
+        if (id) {
+            modelParams.where = { id: id };
+            return this.model.findOne(modelParams);
+        } else {
+            return this.model.findAll(modelParams);
+        }
+    }
+    create(json) {
+        return this.model.create(json)
+    }
+
+    async update(id, json) {
+        let row = await this.model.findOne({ where: { id: id, } });
+        let updatedRow = await row.update(json);
+        return updatedRow;
+    }
+    delete(id) {
+        return this.model.destroy({ where: { id: id } });
+    }
+
```

Comment:
add a route function that take `app` and `name`. In the function it does CRUD functionality, similar to your commented out routes